scripts/astyle.options

```diff
@@ -0,0 +1,11 @@
+--convert-tabs
+--lineend=linux
+--indent=spaces=4
+--style=kr
+--add-brackets
+--max-code-length=80
+--break-after-logical
+--pad-header
+--pad-oper
+--unpad-paren
+--suffix=none
```

Comment:
@rouault Where does theses options come from ? Are they copy-pasted from QGIS ?
@malaterre @mayeut Any comment on the astyle config before we merge the PR ?